scipy/fftpack/tests/test_real_transforms.py

```diff
@@ -814,8 +814,7 @@ def test_axes_and_shape(self):
             assert_raises(ValueError, fforward, self.data,
                           shape=self.data.shape,
                           axes=(0, ))
-            # shape must be a tuple
-            assert_raises(TypeError, fforward, self.data,
+            assert_raises(ValueError, fforward, self.data,
```

Comment:
instead of getting rid of the comment that tells us what the error is, let's modernize this with:
```
with pytest.raises(ValueError, match='some regex that makes the expected error clear'):
    fforword(self.data, shape=self.data.shape[0], axes=(0, 1))
```

Reply:
Done.

Reply:
Not pushed yet?

Reply:
Sorry, I made a mistake with git.  It should be up now.